x-pack/test/functional/apps/security/role_mappings.ts

```diff
@@ -81,7 +81,8 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     });
 
     it('allows a role mapping to be deleted', async () => {
-      await testSubjects.click(`deleteRoleMappingButton-new_role_mapping`);
+      await testSubjects.click('euiCollapsedItemActionsButton');
```

Comment:
I do not think it is really needed because it is the same actions button on the row or in the popover menu item. In my opinion, it should already being handle by the EUI library ;) (We need to trust our tools)